docs/data-sources.md

```diff
@@ -43,34 +66,34 @@ zoom   | _0°_       | _45°_      | _60°_
+**Note:** Esri has [documentation](https://blogs.esri.com/esri/arcgis/2009/03/19/how-can-you-tell-what-map-scales-are-shown-for-online-maps/) about converting web map zoom integers to conventional map scales.
 
 ## What is sourced at what zooms?
 
-Generally speaking, **GMTED** is used at low-zooms, and **ETOPO1** is used to show ocean bathymetry at all zooms (even bathymetry oversampled at zoom 15), and **[SRTM](http://www2.jpl.nasa.gov/srtm/)** is relied on in mid- and high-zooms on land. In the United States, data from **USGS** supplements the SRTM land data to provide greater detail at 10 meters and in some areas at up to 3 meters.
+Generally speaking, **GMTED** is used at low-zooms, **ETOPO1** is used to show ocean bathymetry at all zooms (even bathymetry oversampled at zoom 15), and **[SRTM](http://www2.jpl.nasa.gov/srtm/)** is relied on in mid- and high-zooms on land. Some countries have higher resolution data available over land sourced from other open datasets. More information about these sources is available below.
 
 It should be noted that both `SRTM` and `GMTED` fill oceans and other bodies of water with a value of zero to indicate mean sea level; in these areas, `ETOPO1` provides bathymetry (as well as in regions which are not covered by `SRTM` and `GMTED`).
 
 **Data sources per zoom:**
 
 zoom   | ocean    | land
 ------ | -------- | -------
-**0**  | `ETOPO1` | `GMTED`
-**1**  | `ETOPO1` | `GMTED`
-**2**  | `ETOPO1` | `GMTED`
-**3**  | `ETOPO1` | `GMTED`
+**0**  | `ETOPO1` | `ETOPO1`
+**1**  | `ETOPO1` | `ETOPO1`
+**2**  | `ETOPO1` | `ETOPO1`
+**3**  | `ETOPO1` | `ETOPO1`
 **4**  | `ETOPO1` | `GMTED`
 **5**  | `ETOPO1` | `GMTED`
 **6**  | `ETOPO1` | `GMTED`
-**7**  | `ETOPO1` | `GMTED`
-**8**  | `ETOPO1` | `GMTED`
-**9**  | `ETOPO1` | `SRTM` with `GMTED` in high latitudes above 60°
-**10** | `ETOPO1` | `SRTM` with `GMTED` in high latitudes above 60°
-**11** | `ETOPO1` | `SRTM` with `GMTED` in high latitudes above 60°
-**12** | `ETOPO1` | `SRTM` with `GMTED` and `NED/3DEP` in USA (10 meter)
-**13** | `ETOPO1` | `SRTM` with `GMTED` and `NED/3DEP` in USA (10 meter)
-**14** | `ETOPO1` | `SRTM` with `GMTED` and `NED/3DEP` in USA (mostly 10 meter, some 3 meter)
-**15** | `ETOPO1` | `SRTM` with `GMTED` and `NED/3DEP` in USA (mostly 10 meter, some 3 meter)
+**7**  | `ETOPO1` | `SRTM`, `NRCAN` in Canada, with `GMTED` in high latitudes above 60°
+**8**  | `ETOPO1` | `SRTM`, `NRCAN` in Canada, with `GMTED` in high latitudes above 60°
+**9**  | `ETOPO1` | `SRTM`, `NRCAN` in Canada, `EUDEM` in Europe, with `GMTED` in high latitudes above 60°
+**10** | `ETOPO1`, `3DEP` in California | `SRTM`, `data.gov.at` in Austria, `NRCAN` in Canada, `SRTM`, `NED/3DEP` 1/3 arcsec, `data.gov.uk` in United Kingdom, `INEGI` in Mexico, `ArcticDEM` in latitudes above 60°, `LINZ` in New Zealand, `Kartverket` in Norway
```

Comment:
Why is 3DEP only listed for California versus all the USA?

Reply:
The "in California" column is specific to bathymetry. We don't have any bathy from 3DEP anywhere else.

Reply:
Do the non-California sources all ready ned/3dep in the footprint databases, but are 10 meters or 3 meters?

Reply:
The only sources we have with bathymetry are etopo1 and `ned_topobathy` (which only covers California).

Reply:
Should we just say `ned_topobathy` then – what would match the footprints DB names?